contracts/mocks/ERC721Mock.sol

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity >=0.8.0;
+
+import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/utils/Counters.sol";
+
+contract ERC721Mock is ERC721, Ownable {
```

Comment:
We also have a mock for ERC721s, I think it suits the use case too. So let's use the already existent instead, please
https://github.com/lens-protocol/modules/blob/master/contracts/mocks/NFT.sol